src/profile/helpers/__tests__/updateMutationVariables.test.ts

```diff
@@ -175,30 +162,13 @@ test('update arrays are formed correctly', () => {
   );
 
   expect(variables.input.profile.addAddresses).toEqual([
```

Comment:
Not related to this PR in any way, but should this be checking a field named `updateAddresses`?

Reply:
Good catch. That made no sense and is now fixed.